mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
         name: 'inspectTemplateName'
         description: The name of the template to run when this job is triggered.
         required: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'inspectConfig'
+        description: The core content of the template.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'excludeInfoTypes'
+            description: When true, excludes type information of the findings.
+          - !ruby/object:Api::Type::Boolean
+            name: 'includeQuote'
+            description: When true, a contextual quote from the data that triggered a
+              finding is included in the response.
+          - !ruby/object:Api::Type::Enum
+            name: 'minLikelihood'
+            description: |
+              Only returns findings equal or above this threshold. See https://cloud.google.com/dlp/docs/likelihood for more info
+            values:
+              - :VERY_UNLIKELY
+              - :UNLIKELY
+              - :POSSIBLE
+              - :LIKELY
+              - :VERY_LIKELY
+            default_value: :POSSIBLE
+          - !ruby/object:Api::Type::NestedObject
+            name: 'limits'
+            description: Configuration to control the number of findings returned.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'maxFindingsPerItem'
+                description: Max number of findings that will be returned for each item
+                  scanned. The maximum returned is 2000.
+                required: true
+              - !ruby/object:Api::Type::Integer
+                name: 'maxFindingsPerRequest'
+                description: Max number of findings that will be returned per request/job.
+                  The maximum returned is 2000.
+                required: true
+              - !ruby/object:Api::Type::Array
+                name: 'maxFindingsPerInfoType'
+                description: |
+                  Configuration of findings limit given for specified infoTypes.
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'infoType'
+                      required: true
```

Comment:
Just want to double check if this field is required.

Reply:
I checked this field manually along with `maxFindings` and both aren't required. But as they are part of array, I couldn't add `AtleastOneOf`. And keeping the block empty works, won't be passed in POST call.